sympy/core/tests/test_args.py

```diff
@@ -1054,10 +1054,10 @@ def test_sympy__stats__joint_rv__JointDistributionHandmade():
 
 
 def test_sympy__stats__joint_rv__MarginalDistribution():
-    from sympy.stats.rv import RandomSymbol
+    from sympy.stats import Normal
     from sympy.stats.joint_rv import MarginalDistribution
-    r = RandomSymbol(S('r'))
-    assert _test_args(MarginalDistribution(r, (r,)))
+    N1 = Normal('N1', [1, 0], [[1, 0], [0, 1]])
+    assert _test_args(MarginalDistribution(N1, (N1[1],)))
```

Comment:
I would prefer to construct `MarginalDistribution` without occurrences of `RandomSymbol` in it.
Like:
```python
dist = N1.pspace.distribution
_test_args(MarginalDistribution(dist, (1,)))
```
Distribution objects should not contain `RandomSymbol` or `PSpace` objects, only the parameters defining the distribution.